frontend/javascripts/oxalis/model/helpers/nml_helpers.js

```diff
@@ -167,12 +179,39 @@ function serializeMetaInformation(
   ]);
 }
 
-function serializeBoundingBox(bb: ?BoundingBoxType, name: string): string {
-  const serverBoundingBox = convertFrontendBoundingBoxToServer(bb);
-  if (serverBoundingBox != null) {
-    const { topLeft, width, height, depth } = serverBoundingBox;
-    const [topLeftX, topLeftY, topLeftZ] = topLeft;
-    return serializeTag(name, { topLeftX, topLeftY, topLeftZ, width, height, depth });
+function serializeTaskBoundingBox(boundingBox: ?BoundingBoxType, tagName: string): string {
+  if (boundingBox) {
+    const boundingBoxArray = computeArrayFromBoundingBox(boundingBox);
+    const [topLeftX, topLeftY, topLeftZ, width, height, depth] = boundingBoxArray;
+    return serializeTag(tagName, {
+      topLeftX,
+      topLeftY,
+      topLeftZ,
+      width,
+      height,
+      depth,
+    });
+  }
+  return "";
+}
+
+function serializeUserBoundingBox(bb: UserBoundingBox, tagName: string): string {
+  const { boundingBox, id, name } = bb;
+  const boundingBoxArray = computeArrayFromBoundingBox(boundingBox);
+  if (boundingBoxArray != null && bb) {
+    const [topLeftX, topLeftY, topLeftZ, width, height, depth] = boundingBoxArray;
+    const color = bb.color ? mapColorToComponents(bb.color) : {};
```

Comment:
```suggestion
    const color = bb.color != null ? mapColorToComponents(bb.color) : {};
```

Reply:
Same here: A `Vector3` cannot be evaluated as false unless it is `null` or `undefined`

Reply:
Yes you're right, I've just grown accustomed to always doing it that way, feel free to ignore, I won't include it in future code reviews :)